storage_helper.cpp

```diff
@@ -171,6 +171,32 @@ int StorageHelper::reformat_storage(void) {
     return status;
 }
 
+int StorageHelper::format(FileSystem *fs, BlockDevice *bd) {
+    if (!fs || !bd) return -1;
+
+    int status;
+
+    status = bd->init();
+    if (status != 0) {
+        return status;
+    }
+
+    status = fs->mount(bd);
```

Comment:
Sometimes doing mount on corrupted filesystem would cause hardfault. Perhaps this should be the whole reformat/erase logic. What do you think @janjongboom ?